.github/workflows/build.yaml

```diff
@@ -0,0 +1,31 @@
+name: Tests build
+
+on: [push, pull_request]
+
+env:
+  DVC_TEST: "true"
+  HOMEBREW_NO_AUTO_UPDATE: 1
+  SHELL: /bin/bash
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-2019, macos-10.15]
+        pyv: [3.6, 3.7, 3.8]
```

Comment:
THis looks great. So we can get rid of some of our ugly scripts/ci, right? Doesn't look like you use them at all, actually.

Reply:
I'll get rid of them, just trying to make it work now